src/Internal/Service/ExampleService.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Class ExampleService
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Internal\Service;
+
+use WCPay\Core\Mode;
+
+/**
+ * This is a service, which will be used for developing
+ * DI-related functionality until there are better services
+ * to test with.
+ */
+class ExampleService {
```

Comment:
Yes, I have it in mind. As soon as there are a couple of real-world classes to use, they will replace the example service. I'm confident that it will happen within the payment process re-engineering project.